SharedPackages/BrowserServicesKit/Sources/Networking/Auth/OAuthClient.swift

```diff
@@ -202,20 +200,21 @@ final public actor DefaultOAuthClient: @preconcurrency OAuthClient {
     // MARK: - Public
 
     public var isUserAuthenticated: Bool {
-        tokenStorage.tokenContainer != nil
+        let tokenContainer = try? tokenStorage.getTokenContainer()
+        return tokenContainer != nil
     }
 
     public var currentTokenContainer: TokenContainer? {
         get  {
-            tokenStorage.tokenContainer
+            try? tokenStorage.getTokenContainer()
         }
         set {
-            tokenStorage.tokenContainer = newValue
+            try? tokenStorage.saveTokenContainer(newValue)
```

Comment:
Same though as above.  No need to change anything here, but want to highlight it for future consideration.

Reply:
this is easy, I missed it, changed